test/basic.html

```diff
@@ -39,6 +39,13 @@
           b1 = fixture('Basic');
         });
 
+        test('set scroll direction', function(done) {
+          flush(function() {
+            assert.equal(b1.__polymerGesturesTouchAction, 'pan-y');
```

Comment:
I wish this didn't require inspecting this property but I don't see how to do it otherwise. Seems like there should be a `Polymer.Gestures.getScrollDirection` function.